packages/components/input/src/input.vue

```diff
@@ -6,6 +6,7 @@
       {
         'is-disabled': inputDisabled,
         'is-exceed': inputExceed,
+        'is-hidden': type === 'hidden',
```

Comment:
**Suggestion:** use `v-show` to keep the change minimal.

Reply:
Thank you for your suggestion, I hear someone said `v-show` is going to be deprecated, but I find no description in the doc of Vue.  
Now using `v-show` instead of CSS class.